wgpu/src/lib.rs

```diff
@@ -4739,7 +4765,42 @@ impl<'a> Drop for RenderPass<'a> {
     }
 }
 
-impl ComputePass {
+impl<'a> ComputePass<'a> {
```

Comment:
How about using the same lifetime name everywhere? This helps readers keep track of the role the lifetime is playing.

```suggestion
impl<'pass> ComputePass<'pass> {
```